bower.json

```diff
@@ -12,7 +13,7 @@
     "angular-ui-utils": "bower",
     "angular-ui-router": "~0.2",
     "angular-file-upload": "1.1.5",
-    "angular-messages": "1.3.17",
+    "angular-messages": "~1.3",
```

Comment:
I don't mind, tho this goes quite much into bikeshedding category anyway. :-)

Angular-messages is a core module, so it should be pretty solid and reliable for minor 1.3.x updates, just like the rest of the angular- packages listed there. Just sayin'.